docs/src/man/plotting.md

```diff
@@ -111,3 +111,92 @@ Some special forms of `plot` exist for quickly generating 2d plots of functions.
 # E.g.
 plot([sin, cos], 0, 25)
 ```
+
+## Plotting wide-formatted data
+
+Gadfly is designed to plot data is so-called "long form", in which data that
+is of the same type, or measuring the same quantity, are stored in a single
+column, and any factors or groups are specified by additional columns. This
+is how data is typically stored in a database.
+
+Sometimes data tables are organized by grouping values of the same type into
+multiple columns, with a column name used to distinguish the grouping. We
+refer to this as "wide form" data.
+
+To illustrate the difference consider some historical London birth rate data.
+
+```
+births = RDatasets.dataset("HistData", "Arbuthnot")[[:Year, :Males, :Females]]
+```
+
+ | Row | Year | Males | Females |
+ |-----|------|-------|---------|
+ | 1   | 1629 | 5218  | 4683    |
+ | 2   | 1630 | 4858  | 4457    |
+ | 3   | 1631 | 4422  | 4102    |
+ | 4   | 1632 | 4994  | 4590    |
+ | 5   | 1633 | 5158  | 4839    |
+ | 6   | 1634 | 5035  | 4820    |
+
+This table is wide form because "Males" and "Females" are two columns both
+measuring number of births. Wide form data can always be transformed to long
+form, e.g. with the `stack` function in DataFrames, but this can be
+inconvenient, especially if the data is not already in a DataFrame.
+
+```julia
+stack(births, [:Males, :Females])
+```
+
+ | Row | variable | value | Year |
+ |-----|----------|-------|------|
+ | 1   | Males    | 5218  | 1629 |
+ | 2   | Males    | 4858  | 1630 |
+ | 3   | Males    | 4422  | 1631 |
+ | 4   | Males    | 4994  | 1632 |
+ | 5   | Males    | 5158  | 1633 |
+ | 6   | Males    | 5035  | 1634 |
+
```

Comment:
i'd suggest showing the bottom of this table so that readers don't wonder where all the Females have gone to.

```
│ Row │ variable │ value │ Year │
├─────┼──────────┼───────┼──────┤
│ 1   │ Males    │ 5218  │ 1629 │
│ 2   │ Males    │ 4858  │ 1630 │
│ 3   │ Males    │ 4422  │ 1631 │
│ 4   │ Males    │ 4994  │ 1632 │
│ 5   │ Males    │ 5158  │ 1633 │
⋮
│ 160 │ Females  │ 7417  │ 1706 │
│ 161 │ Females  │ 7687  │ 1707 │
│ 162 │ Females  │ 7623  │ 1708 │
│ 163 │ Females  │ 7380  │ 1709 │
│ 164 │ Females  │ 7288  │ 1710 │
```

would probably also want to show the bottom of the previous table too.